src/content/docs/en/guides/deploy/deno.mdx

```diff
@@ -173,5 +173,16 @@ If your project is stored on GitHub, the [Deno Deploy website](https://dash.deno
 
 <ReadMore>Read more about [SSR in Astro](/en/guides/server-side-rendering/).</ReadMore>
 
+#### Deploy Astro static site with CLI deployment
+1. build the project so that the contents of the `dist` folder are up to date.
+2. In the terminal cd to the `./dist` folder and run the deploy command with the following parameters
+
+```sh
+deployctl deploy --project=<MY-DENO-PROJECT> --entrypoint=https://deno.land/std@0.211.0/http/file_server.ts
```

Comment:
Do this file server behave as an Astro project would expect? The key behaviors that normally need to be configured on static deployments without an adapter to do the work are:
- Folder indexes are handled the same way as Astro, so it matches what Astro generates (remember that this is configurable in Astro)
- Trailing slashes match Astro's configuration
- 404 and 500 pages are handled correctly

Taking as an example NGINX. You clearly can host any static site with it, but if you try to host an Astro site with the default configuration it will misbehave. There is [a section on the docs](https://docs.astro.build/en/recipes/docker/#nginx) with a configuration that allows serving an Astro static output correctly. Would an example like that be needed here or does it work out of the box?

Reply:
Thanks for the review @Fryuni 
Sorry for the late reply. It's been a hectic couple of days. 

I tested the 404 scenario you mentioned and it's indeed an issue. When deployed `404.astro` or `404.md` is not recognised. 

What's the best approach? Shall I close the PR until I have a something more complete that addresses the issue?

Reply:
If you know of a way to configure that file server to serve the 404 correctly, you can include it as the required configuration for that option.

If you don't know of a way, or if you know that there isn't one, then I believe it is better not to make this change.